cellar_rebalancer/src/commands/start.rs

```diff
@@ -3,24 +3,24 @@
 /// App-local prelude includes `app_reader()`/`app_writer()`/`app_config()`
 /// accessors along with logging macros. Customize as you see fit.
 use crate::prelude::*;
-use abscissa_core::error::BoxError;
-use futures::future;
-use signatory::FsKeyStore;
-use std::result::Result;
+
```

Comment:
Nit: This whole import set is a real mix-and-match of package and crate imports, they could use re-ordering.